nvflare/fuel/utils/component_classes.json

```diff
@@ -0,0 +1 @@
ck.aggr_servicer", "MockClientRunner": "nvflare.app_opt.xgboost.histogram_based_v2.mock.mock_client_runner", "MockXGBController": "nvflare.app_opt.xgboost.histogram_based_v2.mock.mock_controller", "MockDataConverter": "nvflare.app_opt.xgboost.histogram_based_v2.mock.mock_data_converter", "TupleAggregator": "nvflare.app_opt.xgboost.histogram_based_v2.mock.mock_data_converter", "MockXGBExecutor": "nvflare.app_opt.xgboost.histogram_based_v2.mock.mock_executor", "MockSecureClientRunner": "nvflare.app_opt.xgboost.histogram_based_v2.mock.mock_secure_client_runner", "MockSecureXGBExecutor": "nvflare.app_opt.xgboost.histogram_based_v2.mock.mock_secure_executor", "MockServerRunner": "nvflare.app_opt.xgboost.histogram_based_v2.mock.mock_server_runner", "Federated": "nvflare.app_opt.xgboost.histogram_based_v2.proto.federated_pb2_grpc", "FederatedServicer": "nvflare.app_opt.xgboost.histogram_based_v2.proto.federated_pb2_grpc", "FederatedStub": "nvflare.app_opt.xgboost.histogram_based_v2.proto.federated_pb2_grpc", "AppRunner": "nvflare.app_opt.xgboost.histogram_based_v2.runners.xgb_runner", "XGBServerRunner": "nvflare.app_opt.xgboost.histogram_based_v2.runners.xgb_server_runner", "ClientSecurityHandler": "nvflare.app_opt.xgboost.histogram_based_v2.sec.client_handler", "DamDecoder": "nvflare.app_opt.xgboost.histogram_based_v2.sec.dam", "DamEncoder": "nvflare.app_opt.xgboost.histogram_based_v2.sec.dam", "AggregationContext": "nvflare.app_opt.xgboost.histogram_based_v2.sec.data_converter", "DataConverter": "nvflare.app_opt.xgboost.histogram_based_v2.sec.data_converter", "FeatureAggregationResult": "nvflare.app_opt.xgboost.histogram_based_v2.sec.data_converter", "FeatureContext": "nvflare.app_opt.xgboost.histogram_based_v2.sec.data_converter", "Adder": "nvflare.app_opt.xgboost.histogram_based_v2.sec.partial_he.adder", "Decrypter": "nvflare.app_opt.xgboost.histogram_based_v2.sec.partial_he.decrypter", "Encryptor": "nvflare.app_opt.xgboost.histogram_based_v2.sec.partial_he.encryptor", "ProcessorDataConverter": "nvflare.app_opt.xgboost.histogram_based_v2.sec.processor_data_converter", "SecurityHandler": "nvflare.app_opt.xgboost.histogram_based_v2.sec.sec_handler", "ServerSecurityHandler": "nvflare.app_opt.xgboost.histogram_based_v2.sec.server_handler", "MetricsCallback": "nvflare.app_opt.xgboost.metrics_cb", "XGBBaggingAggregator": "nvflare.app_opt.xgboost.tree_based.bagging_aggregator", "XGBModelPersistor": "nvflare.app_opt.xgboost.tree_based.model_persistor", "XGBModelShareableGenerator": "nvflare.app_opt.xgboost.tree_based.shareable_generator", "CallInfo": "nvflare.widgets.comp_caller", "ComponentCaller": "nvflare.widgets.comp_caller", "ClientFedEventRunner": "nvflare.widgets.fed_event", "FedEventRunner": "nvflare.widgets.fed_event", "ServerFedEventRunner": "nvflare.widgets.fed_event", "GroupInfoCollector": "nvflare.widgets.info_collector", "InfoCollector": "nvflare.widgets.info_collector", "Widget": "nvflare.widgets.widget", "WidgetID": "nvflare.widgets.widget"}
```

Comment:
This is a snapshot of all Flare components that can support the "name" specification. Since this is generated only once offline, we can include everything. In run time, these will be loaded without scanning again.